bot/cogs/eggs/reaction.py

```diff
@@ -0,0 +1,27 @@
+from discord.ext.commands import Cog
+from datetime import datetime, timedelta
+
+
```

Comment:
Personally, I'd suggest leaving just a single empty line, unless this is a Python convention.